test/e2e_federation/framework/framework.go

```diff
@@ -137,10 +137,6 @@ func (f *Framework) FederationAfterEach() {
 		framework.DumpEventsInNamespace(func(opts v1.ListOptions, ns string) (*v1.EventList, error) {
 			return f.FederationClientset.Core().Events(ns).List(opts)
 		}, f.FederationNamespace.Name)
-		// Print logs of federation control plane pods (federation-apiserver and federation-controller-manager)
-		framework.LogPodsWithLabels(f.ClientSet, "federation", map[string]string{"app": "federated-cluster"}, framework.Logf)
```

Comment:
@madhusudancs, Basically, the f.ClientSet is from the e2e framework and this is set to nil in framework AfterEach which gets called before FedearationAfterEach.